<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <disabledon>../New folder (10)/وکتور-طرح-انتزاعی-نقشه-زمین.jpg</disabledon>
    <activeon>../New folder (10)/وکتور-طرح-انتزاعی-نقشه-زمین.jpg</activeon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69))</string>
  </property>
  <property name="iconSize">
   <size>
    <width>150</width>
    <height>138</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Algerian</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>world_clock</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Digital-7 Italic</family>
              <pointsize>72</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>  00:00 </string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Digital-7 Italic</family>
              <pointsize>72</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string>  00:00 </string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Digital-7 Italic</family>
              <pointsize>72</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>  00:00 </string>
            </property>
            <property name="cursorPosition">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Digital-7 Italic</family>
              <pointsize>72</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>  00:00 </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="font">
             <font>
              <family>Digital-7 Italic</family>
              <pointsize>72</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>WORLD</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="font">
             <font>
              <family>Digital-7 Italic</family>
              <pointsize>72</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>US</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="font">
             <font>
              <family>Digital-7 Italic</family>
              <pointsize>72</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>IRAN</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="font">
             <font>
              <family>Digital-7 Italic</family>
              <pointsize>72</pointsize>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>GERMANY</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>ALARM</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_8">
          <property name="font">
           <font>
            <family>Digital-7 Italic</family>
            <pointsize>36</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_5">
          <property name="font">
           <font>
            <family>Digital-7 Italic</family>
            <pointsize>36</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_6"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>STOP_WATCH</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_6">
          <property name="font">
           <font>
            <family>Digital-7 Italic</family>
            <pointsize>28</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>START</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_7">
          <property name="font">
           <font>
            <family>Digital-7 Italic</family>
            <pointsize>28</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>STOP</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="font">
           <font>
            <family>Digital-7 Italic</family>
            <pointsize>72</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>          00:00:00</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>TIMER</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_10">
          <property name="font">
           <font>
            <family>Digital-7 Italic</family>
            <pointsize>28</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>STOP</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_9">
          <property name="font">
           <font>
            <family>Digital-7 Italic</family>
            <pointsize>28</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>START</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="lineEdit_6">
          <property name="font">
           <font>
            <family>Digital-7 Italic</family>
            <pointsize>72</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>          00:00:00</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
